src/stylesheets/components/Products/GridItem.sass

```diff
@@ -1,50 +1,50 @@
 .gridItem-container
-  @include flex
+    @include flex
 
 .gridItem
-  $root: &
-  @include flex($direction: column)
-  cursor: pointer
-  margin: 0 0 $marg-4 0
-  position: relative
-  width: 250px
-
-  &:hover img
-    transform: scale(1.1) rotate(5deg)
-
-  &:hover #{$root}__show-icon,
-  &:hover #{$root}__show-button
-    display: block
-
-  &__img-wrap
-    overflow: hidden
-
-  &__description
-    margin-top: $marg-base
-    text-align: center
-
-  &__text
-    font-family: Montserrat
-    ont-size: $font-size-base
-    font-weight: 500
-    line-height: 24px
-
-  &__first-icon
-    position: absolute
-    right: $marg-base
-    top: $marg-base
-
-  &__show-icon
-    animation: fadeInShort 0.6s ease
-    display: none
-    position: absolute
-    right: $marg-base
-    top: $marg-6
-
-  &__show-button
-    animation: fadeInShort 0.6s ease
-    bottom: 80px
-    display: none
-    position: absolute
-    text-align: center
-    width: 100%
+    $root: &
+    @include flex($direction: column)
+    cursor: pointer
+    margin: 0 0 $marg-4 0
+    position: relative
+    width: 250px
+
+    &:hover img
+        transform: scale(1.1) rotate(5deg)
+
+    &:hover #{$root}__show-icon,
+    &:hover #{$root}__show-button
+        display: block
+
+    &__img-wrap
+        overflow: hidden
+
+    &__description
+        margin-top: $marg-base
+        text-align: center
+
+    &__text
+        font-family: Montserrat
```

Comment:
This is more design-related, but the 3 fonts that you're using: Montserrat, Open Sans and Raleway should not be used together in one project as the differences between them are minimal. It's best to use one font-family with multiple weights